contents/docs/integrate/user-properties.mdx

```diff
@@ -5,41 +5,92 @@ sidebar: Docs
-
-Sometimes we might want to mix `set` and `set_once` usage. Imagine, when we have some heuristics that help us determine a value, but users can also specify it. In these cases it might be beneficial to use `set_once` for the heuristically computed value and `set` for user specified value. Note that we will never override an existing value if `set_once` is used, so if heuristics are used on a continuous basis that would not be a good strategy.
+Use `set` when we want the value should always be the newest one. For example, to update the email for a user.
 
-## How do values get overridden?
+Use `set_once` when we want to set the first value and it to never be updated afterwards, e.g. the first URL that referred a user to the site.
 
-`set` always overrides, `set_once` only writes when the value doesn't exist.
+In summary: `set` always overrides, `set_once` only writes when the property doesn't already exist on the user.
 
-`identify` and `alias` upon name conflict use the value the first user provided. This series of calls would mean we end up with `location` being `Rome`:
-```python
-posthog.capture('anon', event='set location', properties={'$set': {'location': 'Paris'}})
-posthog.capture('Alice', event='set location', properties={'$set_once': {'location': 'Rome'}})
-posthog.alias('Alice', 'anon')
-```
-
-Note that we ignore the event timestamps and just process everything at ingestion time.
+For example:
+```js
+posthog.people.set( { location: 'London'  })
+posthog.people.set( { location: 'Rome'  })
 
+// { location: 'Rome' }
 
+posthog.people.set_once( { initial_location: 'London'  })
+posthog.people.set_once( { initial_location: 'Rome'  })
 
+// { location: 'London' }
+```
 
+Sometimes we might want to use both `set` and `set_once` usage. Imagine, when we have some heuristics that help us determine a value, but users can also specify it. In these cases it might be beneficial to use `set_once` for the heuristically computed value and `set` for user specified value. This way we can ensure that the user specified value is always used, but if the user doesn't specify it, we can still use the heuristically computed value.
 
+## How are properties managed when merging or aliasing users
 
+// TODO: Confirm this with Tiina. Is this what this section means: https://posthog.com/docs/integrate/user-properties#how-do-values-get-overridden
 
+When two users both contain a property and are merged, the property of the first user (i.e. the distinctId for the alias) are used.
+```js
+// this uses the node sdk
+client.identify({
+    distinctId: 'main_id',
+    properties: {
+        location: 'London',
+    },
+})
+
+client.identify({
+    distinctId: 'secondary_id',
+    properties: {
+        location: 'Rome',
+    },
+})
+
+posthog.alias({
+    distinctId: 'main_id',
+    alias: 'secondary_id',
+})
+
+// for user 'main_id', { location: 'London' }
+```
```

Comment:
@tiina303 can you help me with this section? This is my guess of what it should be. 

I can't understand the original: https://posthog.com/docs/integrate/user-properties#how-do-values-get-overridden

> set always overrides, set_once only writes when the value doesn't exist.
> 
> identify and alias upon name conflict use the value the first user provided. This series of calls would mean we end up with location being Rome:
> 
> ```Python
> posthog.capture('anon', event='set location', properties={'$set': {'location': 'Paris'}})
> posthog.capture('Alice', event='set location', properties={'$set_once': {'location': 'Rome'}})
> posthog.alias('Alice', 'anon')
> ```
> Note that we ignore the event timestamps and just process everything at ingestion time.
>